doc/contributing/contributing.rst

```diff
@@ -31,49 +31,55 @@ To contribute to the source code of the model you will need to fork the reposito
 Quickstart Guide
 ----------------
 
-1. Fork the project on GitHub (using the fork button) and create a local clone, if you haven’t already:
+**1.** Fork the project on GitHub (using the fork button).
+
+**2.** Create a local clone (we strongly suggest keeping a separate repository for development work):
```

Comment:
It seems unnecessary and confusing to say "we strongly suggest keeping a separate repository for development work". This guide is for contributors. There is literally no other way to contribute.

Reply:
This is intended to steer people away from using the same set of local files for both running the model for science and developing the model. 

We could rephrase to "...keeping a separate set of local files for development work."